### ISSUE TITLE
Water is unworkable

### ISSUE THREAD

**Issue by Fl4shblade** [ferram4/Ferram-Aerospace-Research#160]
@ferram4 You specifically asked for feedback on water drag as I recall so here it is. It is basically what the topic says but I will get into more detail of course. I took the stock gull to the water and at first I experienced something strange. The plane slipped into the water at about 10 m/s and at zero thrust there remained about 6.5 m/s that would not decelerate. I had to rotate the plane nearly 360 degrees to get rid of that phantom speed. I have 100% take-off failure from the water. At first the plane seems to accelerate just fine, but at about 40 m/s at the lowest and 50 m/s at the highest the water resistance increases like crazy, tearing of the skids on the wings and bringing the plane to a screaching halt in an instant. Compared to stock where the plane can just barely reach 41 m/s to take-off from the water but at about 30 m/s the plane is calmly increasing speed and the water doesn't seem much of a hinderance. I also tried a water landing at about 50 m/s which was the lowest speed I could manage with the gull since it has no air-brakes in stock config. Again it is basically insta-deceleration and the planes nose dove under the water-line and flipped on the back. What seems to be missing here is that with increased speed there is ever more of a skidding over the waters surface happening in reality. With FAR or stock this lifting out of the water never seems to take place and the parts are just as partially below the surface line at 0 speed as they are at high speed. There is also this visual cloud effect when you accelerate in water with FAR that seems to either not be there or is less much visible compared to stock when you accelerate in the water.

**Comment by ferram4** [comment]
The plane not decelerating is not a bug, it is simply very low deceleration due to the very low speed.  I can increase the drag at very low speeds drastically, but I currently don't have any data on how that should be modeled, especially given the perfectly flat water surface KSP has.

For take off, this does not sound like a bug, this sounds like proper behavior.  40 m/s is 144 km/h and 90 mph, both extreme speeds for getting sea planes out of the water and heavy sea planes requiring fast take off  I have only found one sea plane with a stall speed close to that speed, and the pontoons that it has are rather large to compensate.  Never, ever try to compare the flight characteristics of planes designed for stock and assume they should behave properly in FAR.

So basically, thanks for confirming that the only minor issue with water drag that exists is one that I was already aware of.  It's unlikely that anything needs to be changed then.
